chia/rpc/data_layer_rpc_api.py

```diff
@@ -428,3 +429,10 @@ async def get_sync_status(self, request: Dict[str, Any]) -> EndpointResult:
                 "target_generation": sync_status.target_generation,
             }
         }
+
+    async def check_plugins(self, request: Dict[str, Any]) -> EndpointResult:
+        if self.service is None:
+            raise Exception("Data layer not created")
+        plugin_status = await self.service.check_plugins()
+
+        return {"plugin_status": {"uploaders": plugin_status.uploaders, "downloaders": plugin_status.downloaders}}
```

Comment:
Should `PluginStatus` provide some marshalling for this?

Reply:
added